Updated Product feature List documentation (#457)
mojaloop/documentation: docs/product/features/transactions.md

# Mojaloop Transactions
This section deals with all of the aspects of a Mojaloop transaction.
<details>
<summary>**Transaction Phases**</summary>

## Phases of a Mojaloop Transaction

> Mojaloop is a payment clearing hub: it clears (routes and guarantees)

Reviewer: ```suggestion
A payment clearing hub based on a Mojaloop platform clears (routes and guarantees)
```

    of disputed transactions and eliminates exception processing, and so
    substantially reduces costs for all participants.

> 	The Mojaloop community provides a number of tools that can be freely used by DFSPs to connect to a Mojaloop Hub. These remain within the DFSP's domain, and are not the concern of the hub operator or any other party. As well as managing the connection to the Hub and facilitating transactions, these tools also ensure the security of the connection and in particular provide the key DFSP link to this non-repudiation capability.  

Reviewer: ```suggestion
	Mojaloop provides a number of tools that can be freely used by DFSPs to connect to a Mojaloop Hub. These remain within the DFSP's domain, and are not the concern of the hub operator or any other party. As well as managing the connection to the Hub and facilitating transactions, these tools also ensure the security of the connection and in particular provide the key DFSP link to this non-repudiation capability.  
```
